doc/articles/guides/07-dialogs.md

```diff
@@ -0,0 +1,282 @@
+# Dialogs and errors
+
+The Silverlight **TimeEntryRia** makes use of **ChildWindow** controls to display dialog controls such as the **LoginForm** and **ErrorWindow**. The equivalent UWP control for these types of dialogs is the **ContentDialog**.
+
+> [!TIP]
+> Dialog controls are modal UI overlays that provide contextual app information. They block interactions with the app window until being explicitly dismissed. They often request some kind of action from the user. The Microsoft guidelines on dialog controls in UWP can be reviewed below:
+>
+> * [Dialog controls](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs)
+> * [ContentDialog class](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ContentDialog)
+
+The **ContentDialog** is a flexible control a dialog box that can be customized to contain checkboxes, hyperlinks, buttons and any other XAML content. It can be created simply from code, or as a new item added to a project with full designer support. However, there is one major limitation:
```

Comment:
```suggestion
The **ContentDialog** is a flexible control - a dialog box that can be customized to contain checkboxes, hyperlinks, buttons and any other XAML content. It can be created simply from code, or as a new item added to a project with full designer support. However, there is one major limitation:
```